pkg/webhook/tenant/forbidden_annotations_regex.go

```diff
@@ -0,0 +1,75 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
Wrong header here:

```
// Copyright 2020-2021 Clastix Labs
// SPDX-License-Identifier: Apache-2.0
```

Reply:
This didn't seem right to me, but anyway, done